man/dashbioCircos.Rd

```diff
@@ -39,16 +39,75 @@ selectEvent={
     "2": "both"
     },}
 
-\item{layout}{List. The overall layout of the Circos graph, provided
+\item{layout}{List of lists containing elements 'len', 'color', 'label', 'id'.
+those elements have the following types:
+  - len (numeric; required): the length of the block.
+  - color (character; required): the color of the block.
+  - label (character; required): the labels of the block.
+  - id (character; required): the id of the block, where it will recieve
+data from the specified "track" id.s. The overall layout of the Circos graph, provided
```

Comment:
It might be better to have everything after/including "The overall layout..."  before this list -- it kind of looks like it's part of the last element in the list, but it isn't. The extraneous `s.` should be covered by my changes in https://github.com/plotly/dash/pull/782, but those haven't been released yet.

Reply:
Yeah, this is a tricky one (but a very valid point). It's because the generator detects that `layout` has `PropTypes.exact`, and so it handles the nested typing. Because it's a typing argument, it's at the head of the string (as are all types).

I'm not entirely sure how we would break the typing string in half, and insert the _The overall layout of the Circos graph, provided as a list of dictionaries._ after _..., 'id'._ but I can look into it for the next release. I'll open an issue in the `dash` repo so I don't lose track of this.

Reply:
Thanks for the feedback; noted in https://github.com/plotly/dash/issues/799.